common/pipeline_trigger/README.md

```diff
@@ -0,0 +1,153 @@
+<!-- Overview -->
+# Overview
+
+When using Azure Machine Learning(AML) Service, there are cases where it is necessary to operate the pipeline under various business scenario. This sample demonstrates a scenario where users without access to AML can trigger an AML pipeline using an azure function.
+
+**What does this sample demonstrate:**
+- How to invoke AML pipeline using Azure Function
+
+**What doesn't this sample demonstrate**:
+- Azure Machine Learning Service Creation.
+
+![architecture](media/architecture.png)
```

Comment:
If I check this readme with preview, it seems like image is broken! 
![image](https://user-images.githubusercontent.com/16282358/108188093-85254000-7152-11eb-8097-8074aafa345f.png)

Reply:
If you look at it in the branch it seems to work https://github.com/liupeirong/MLOpsManufacturing/tree/aykhara/pipelinetrigger/common/pipeline_trigger